src/ControlSurface/Controls/Colorpicker/Colorpicker.qml

```diff
@@ -0,0 +1,185 @@
+//  A toy QML colorpicker control, by Ruslan Shestopalyuk
+import QtQuick 2.11
+import QtQuick.Layouts 1.1
+import QtQuick.Controls 2.4
+
+//import "content"
+
+Rectangle {
+    id: picker
+    property color colorValue: paletteMode ? _rgb(paletts.paletts_color,
+                                          alphaSlider.value) : _hsla(
+                                         hueSlider.value,
+                                         sbPicker.saturation,
+                                         sbPicker.brightness,
+                                         1)
+    property bool enableAlphaChannel: true
+    property bool enableDetails: true
+    property int colorHandleRadius: 8
+    property bool paletteMode: false
+    property bool enablePaletteMode: false
+    property string switchToColorPickerString: "Palette..."
+    property string switchToPalleteString: "Color Picker..."
+
+    signal colorChanged(color changedColor)
+
+    clip: true
+
+    color: "#363636"
+
+    SBPicker {
+        id: sbPicker
+        anchors.top: parent.top
+        anchors.bottom: parent.bottom
+        anchors.left: parent.left
+        width: colorpicker.width * (83 / 100)
+
+        hueColor: {
+            var v = 1.0 - hueSlider.value
```

Comment:
In JS we tend to prefer `const` and `let` to `var` as they have stricter semantics and can be better optimized